airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -491,25 +491,30 @@ class MetafieldProduct(Metafield):
 class MetafieldProductImage(Metafield):
     """
     {
-        products(query: "updated_at:>='2023-02-07T00:00:00+00:00' AND updated_at:<='2023-12-04T00:00:00+00:00'", sortKey: UPDATED_AT) {
+        products(query: "updated_at:>='2023-01-08T00:00:00+00:00' AND updated_at:<='2024-08-02T15:12:41.689153+00:00'", sortKey: UPDATED_AT) {
```

Comment:
I've just copy/paste the real query as an example.

Yes, the Shopify can handle these types for GraphQL easily:
- 2023-01-01
- 2023-01-01T01:01:01
- 2023-01-01T01:01:01-00:00
- 2023-01-01T01:01:01+00:00
- 2023-01-01T01:01:01Z
- 2023-01-01T01:01:01.231233-00:00
- 2023-01-01T01:01:01.231233+00:00
- 2023-01-01T01:01:01.231233Z

Because they have their own parser on the server side. Lucky we are.